src/main/java/gameboy/gamepack/racinggame/data/entity/Referee.java

```diff
@@ -0,0 +1,43 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.*;
+
+import gameboy.gamepack.racinggame.data.vo.Position;
+import gameboy.gamepack.racinggame.data.vo.RaceLog;
+
+public class Referee {
+
+    private static final int MIN_POSITION_VALUE = 0;
+
+    private RaceRecorder recorder;
+
+    public Referee() {
+        this.recorder = new RaceRecorder();
+    }
+
+    public void record(List<Car> cars) {
+        recorder.record(cars);
+    }
+
+    public List<RaceLog> playback() {
+        return recorder.playback();
+    }
+
+    public List<Car> getWinners() {
+        RaceLog raceLog = recorder.playbackLastLog();
+        Position winnerPosition = getWinnerPosition(raceLog);
+        return raceLog.getCars()
+            .stream()
+            .filter(car -> car.isSamePosition(winnerPosition))
+            .toList();
+    }
+
+    private Position getWinnerPosition(RaceLog raceLog) {
+        return raceLog.getCars()
+            .stream()
+            .map(Car::getPosition)
+            .max(Position::compareTo)
+            .orElse(new Position(MIN_POSITION_VALUE));
```

Comment:
이 부분은 피드백 반영이 안 되어있네요
Position의 기본값을 매번 생성할 필요가 없습니다. 
default 같은 값들은 캐싱해두고 재사용하는 게 맞습니다.
```suggestion
            .orElse(Position.DEFAULT_ZERO);
```

Reply:
orElse와 orElseGet 차이도 공부해 보시면 좋겠습니다.